resources/docs/simulation/concepts/designing-with-process-models/using-data-in-a-process-model.md

```diff
@@ -14,7 +14,7 @@ You can use your real world business data to power your process model and create
 For more on using data in HASH, see [Datasets](/docs/simulation/creating-simulations/datasets/)
 </Hint>
 
-HASH supports CSV and JSON datasets. Upload your dataset by clicking the 'new dataset' option in the editor \(_in the top left of the screen, over the file picker\)_\) _o_r by finding/reusing an existing dataset from the HASH index and adding it to the simulation.
+HASH supports CSV and JSON datasets. Upload your dataset by clicking the 'new dataset' option in the editor \(_in the top left of the screen, over the file picker\)_\) \_o_r by finding/reusing an existing dataset from the HASH index and adding it to the simulation.
```

Comment:
```suggestion
HASH supports CSV and JSON datasets. Upload your dataset by clicking the 'new dataset' option in the editor \(_in the top left of the screen, over the file picker\)_\) or by finding/reusing an existing dataset from the HASH index and adding it to the simulation.
```

Reply:
There's still an errant additional closing bracket here.

Reply:
I’ve already fixed this locally – strange. Let me try to commit your suggestion anyway and see what’s going to happen.

Reply:
Latest source (looks good):

```markdown
HASH supports CSV and JSON datasets. Upload your dataset by clicking the 'new dataset' option in the editor _(in the top left of the screen, over the file picker)_ or by finding/reusing an existing dataset from the HASH index and adding it to the simulation.
```